test/Unittest.py

```diff
@@ -0,0 +1,19 @@
+class Unittest():
+
+    def __init__(self, code, language, is_input=True, is_output=True):
+        self.language = language
+        self.code = code
+        self.is_input = is_input
+        self.is_output = is_output
+
+    def get_lang(self):
+        return self.language
+
+    def get_code(self):
+        return self.code
+
+    def get_input(self):
```

Comment:
Nit: Would prefer `get_is_input`, this read as "get the input code" when I read it the first time.